src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -43,277 +48,1082 @@ public static void ClassCleanup()
+
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = smallWindowSize;
+            Assert.AreEqual("(464, 502)", page.CalculatorApp.GetCalculatorWindowSize());
+
+            page.StandardAoTCalculatorPage.NavigateToStandardAoTMode();
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = largeAoTWindowSize;
+            Assert.AreEqual("(502, 502)", page.CalculatorApp.GetCalculatorWindowSize());
+
+            page.StandardAoTCalculatorPage.NavigateToStandardMode();
+            Assert.AreEqual("(464, 502)", page.CalculatorApp.GetCalculatorWindowSize());
+
+            page.StandardAoTCalculatorPage.NavigateToStandardAoTMode();
+            Assert.AreEqual("(502, 502)", page.CalculatorApp.GetCalculatorWindowSize());
 
-            page.MemoryPanel.MemButton.Click();
-            page.StandardOperators.NumberPad.Input(3.3);
-            page.MemoryPanel.MemRecall.Click();
-            var memoryItems = page.MemoryPanel.GetAllMemoryListViewItems();
-            Assert.IsTrue(memoryItems[0].Text.Equals("4", StringComparison.InvariantCultureIgnoreCase));
         }
 
         [TestMethod]
-        public void Memory_ClearTest()
+        [Priority(2)]
+        public void AoT_ErrorMessage()
         {
-            page.StandardOperators.NumberPad.Input(12);
+            page.StandardAoTCalculatorPage.NavigateToStandardAoTMode();
+            page.StandardAoTCalculatorPage.ResizeAoTWindowToDiplayInvertButton();
             page.StandardOperators.DivideButton.Click();
-            page.StandardOperators.NumberPad.Input(3);
+            page.StandardOperators.NumberPad.Num0Button.Click();
             page.StandardOperators.EqualButton.Click();
+            page.StandardAoTCalculatorPage.AoTModeCheck();
+            Assert.AreEqual("True", page.StandardAoTCalculatorPage.AoTModeCheck());
+            Assert.AreEqual("Result is undefined", page.CalculatorResults.GetAoTCalculatorResultText());
+            page.StandardOperators.ClearButton.Click();
+            page.StandardOperators.InvertButton.Click();
+            page.StandardAoTCalculatorPage.AoTModeCheck();
+            Assert.AreEqual("True", page.StandardAoTCalculatorPage.AoTModeCheck());
+            Assert.AreEqual("Cannot divide by zero", page.CalculatorResults.GetAoTCalculatorResultText());
+            page.StandardAoTCalculatorPage.NavigateToStandardMode();
+            page.StandardAoTCalculatorPage.AoTModeCheck();
+            Assert.AreEqual("False", page.StandardAoTCalculatorPage.AoTModeCheck());
+            Assert.AreEqual("Cannot divide by zero", page.CalculatorResults.GetCalculatorResultText());
+        }
 
-            page.MemoryPanel.MemButton.Click();
-            page.StandardOperators.NumberPad.Input(3.3);
-            page.MemoryPanel.MemoryClear.Click();
-            page.MemoryPanel.OpenMemoryPanel();
-            Assert.IsNotNull(page.MemoryPanel.MemoryPaneEmptyLabel);
+        #endregion
+
+        #region Stress Tests
```

Comment:
Stress tests should be in their own test class. We can then ensure that we are doing a full stress test and run it on demand.

Reply:
Good idea.  To address this idea, I created CalculatorAppStressTests.cs.  This cs will contain this stress test and future stress tests.